.travis.yml

```diff
@@ -11,17 +11,21 @@ virtualenv:
   system_site_packages: true
 
 env:
-  - VERSION="8.0" INCLUDE="broken_module" SERVER_EXPECTED_ERRORS="1"  # test errors are detected
-  - VERSION="master" EXCLUDE="broken_module"  # EXCLUDE usage example - update with erroring modules list
-  - VERSION="8.0" INCLUDE="test_module,second_module" OPTIONS="--log-level=debug"  # OPTIONS usage example
-  - VERSION="8.0" INCLUDE="test_module,second_module" INSTALL_OPTIONS="--log-level=info"
-  - VERSION="8.0" INCLUDE="test_module,second_module" UNIT_TEST="1"
-  - VERSION="7.0" INCLUDE="test_module,second_module" ODOO_REPO="OCA/OCB"  # ODOO_REPO usage example
-  - VERSION="6.1" INCLUDE="test_module,second_module"
-  - VERSION="" INCLUDE="test_module,second_module"  # test fallback to script argument
-  # - VERSION="6.0"  # Test script not ready for v6.0
+  global:
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0"
+
+  matrix:
+  - TESTS="1" INCLUDE="broken_module" SERVER_EXPECTED_ERRORS="1"  # test errors are detected
+  - TESTS="1" VERSION="master" EXCLUDE="broken_module"  # EXCLUDE usage example - update with erroring modules list
+  - TESTS="1" INCLUDE="test_module,second_module" OPTIONS="--log-level=debug"  # OPTIONS usage example
+  - TESTS="1" INCLUDE="test_module,second_module" INSTALL_OPTIONS="--log-level=info"
+  - TESTS="1" INCLUDE="test_module,second_module" UNIT_TEST="1"
+  - TESTS="1" VERSION="7.0" INCLUDE="test_module,second_module" ODOO_REPO="OCA/OCB"  # ODOO_REPO usage example
+  - TESTS="1" VERSION="6.1" INCLUDE="test_module,second_module"
+  - TESTS="1" VERSION="" INCLUDE="test_module,second_module"  # test fallback to script argument
+  # - TESTS="1" VERSION="6.0"  # Test script not ready for v6.0
   - LINT_CHECK="1"
-  - VERSION="7.0" INCLUDE="test_module,second_module" LINT_CHECK="0"
+  - TESTS="1" VERSION="7.0" INCLUDE="test_module,second_module"
```

Comment:
@dreispt TESTS is set to "0" by default

I find convenient to see which lines are test putting this `TESTS="1"` a beginning of line.